content/applications/inventory_and_mrp/inventory/barcode/operations/transfers_scratch.rst

```diff
@@ -1,17 +1,200 @@
+
+To process and scan each product individually, choose a specific product line. The :guilabel:`+#`
+button (in this case, :guilabel:`+1`) can be clicked to add additional quantity of that product to
+the transfer, or the :guilabel:`pencil icon` can be clicked to open a new screen to edit that
+product line.
+
+From this screen, the product that's being transferred is listed. Under the product name, the
+:guilabel:`Quantity` line can be edited. Change the number in the line to the quantity listed to be
+transferred on the internal transfer form.
+
+.. example::
+   In the internal transfer operation `WH/INT/000XX`, `50 Units` of the `Transfer Product` is
+   expected to be transferred. `[TRANSFER_PROD]` is the :guilabel:`Internal Reference` set on the
+   product form. Scan the barcode of the `Transfer Product` to receive one unit. Afterwards, click
+   the :guilabel:`pencil icon` to manually enter the transferred quantities.
+
+   .. image:: transfers_scratch/transfers-scratch-product-line-editor.png
+      :align: center
+      :alt: Product line editor for individual transfer in Barcode app.
+
+Additionally, the :guilabel:`+1` and :guilabel:`-1` buttons can be clicked to add or subtract
+quantity of the product, and the :guilabel:`number keys` can be used to add quantity, as well.
+
+Below the :guilabel:`number keys` are the two :guilabel:`location` lines, which read `WH/Stock` by
+default, or whichever locations were previously specified on the internal transfer form. Click these
+lines to reveal a drop-down menu of additional locations to choose from.
+
+Once ready, click :guilabel:`Confirm` to confirm the changes made to the product line.
+
+Then, from the overview page with all products to process within that transfer (**WH/INT/000XX**),
+click :guilabel:`Validate`. The receipt has now been processed, and the :guilabel:`Barcode app` can
+be closed out.
+
+.. tip::
+   The *Barcode* app can also be used to scan products in internal transfers containing unique lot
+   numbers and serial numbers. From the barcode transfer screen, scan the barcode of a lot or serial
+   number, and Odoo will automatically increase the quantity of the product to the quantity recorded
+   in the database. If the same lot or serial number is shared between different products, scan the
+   product barcode first, then the barcode of the lot/serial number.
+
+Create a transfer from scratch
 ==============================
-Create a Transfer from Scratch
-==============================
 
-To create a transfer from the *Barcode* application, you first need to
-print the operation type barcodes. To do so, you can download the
-*Stock barcode sheet* from the home page of the app.
+In addition to processing and scanning barcodes for existing, previously-created internal transfers,
+the :guilabel:`Barcode app` can also be used to create transfers from scratch, simply by scanning a
+printed operation type barcode.
+
+.. admonition:: Did you know?
```

Comment:
Creative usage of a custom admonition block!!! :D

Reply:
Thank you!!